predictor.js

```diff
@@ -2044,8 +2044,18 @@ const Koviko = {
             newStatisticValue = (state.resources.expectedSS+state.resources.nonDungeonSS) / totalMinutes;
             legend="SS Expected";
           } else if (trackedStat.name=="act") {
-            newStatisticValue= loop / totalMinutes;
-            legend=actions[finalIndex].name;
+            let finalAction = actions[finalIndex];
+            if (finalAction.disabled) {
+              // Find the last non-disabled action
+              for (let i = finalIndex - 1; i >= 0; i--) {
+                if (!actions[i].disabled) {
+                  finalAction = actions[i];
+                  break;
+                }
+              }
+            }
+            newStatisticValue = state.resources.finLoops / totalMinutes;
```

Comment:
I think this is the line that's actually fixing things - using the saved `finLoops` "resource" rather than "the number of loops of the current action". Nice catch.